definitions/ext-host/definition.yml

```diff
@@ -10,9 +10,20 @@ synthesis:
         - attribute: metricName
           prefix: datadog.system.
 
+    # telemetry from generic flex agent
+    - identifier: displayName
+      name: displayName
+      encodeIdentifierInGUID: true
+      conditions:
+      - attribute: eventType
+        value: FlexSystemSample
+
 dashboardTemplates:
-  newRelic:
-    template: dashboard.json
+  flexagent:
```

Comment:
This value needs to exist as the `intrumentation.provider` attribute of the telemetry. would that happen?

I was searching for some data but didn't find any I could use as reference

Reply:
Yes, I've updated the integration to include it.

![image](https://user-images.githubusercontent.com/13043208/144162204-f99f3d47-03a5-4ae7-a6a6-1006fe52d74c.png)

account id: 1061074

Reply:
perfect thanks!